### PR TITLE
incorporate expression parser

### PR DESCRIPTION
**Pull request by tballmsft**

### CODE REVIEW

**Review by pelikhan — APPROVED** [review]

File: services/tests/button.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
 
 ## downUp: press down and up
     
-    say "press the button and release it immediately"
-    ask "did you observe an Up event, followed by a Down event?"
+press the button and release it immediately
+    ask("did you observe an Up event, followed by a Down event?")
```
Comment (pelikhan) [inline services/tests/button.md:6]: you probaby want to support a newline here to make markdown rendering happy

**Comment by pelikhan** [comment]
I've add an editor for these files /tools/service-test/editor.

File: spectool/jsep/jsep.d.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
+//     JavaScript Expression Parser (JSEP) <%= version %>
```
[the remark this answers is not in the document]
Comment (tballmsft) [reply spectool/jsep/jsep.d.ts]: it's not in NPM, AFAIK

[the remark this answers is not in the document]
Comment (tballmsft) [reply spectool/jsep/jsep.d.ts]: oops - yes, it is there...

**Pull request merged**